javascript/src/main/components/Courses/CourseDetail.js

```diff
@@ -0,0 +1,13 @@
+  
+import React from "react";
+import { useParams } from "react-router-dom";
+//placeholder page for tutor detail
```

Comment:
Do we need this placeholder comment?

Reply:
We don't because this comment was there for another story.